python/pyarrow/tests/test_udf.py

```diff
@@ -39,6 +39,30 @@ class MyError(RuntimeError):
     pass
 
 
+@pytest.fixture(scope="session")
+def unary_agg_func_fixture():
+    """
+    Register a unary aggregate function
+    """
+
+    def func(ctx, x):
+        return pa.array([len(x)])
```

Comment:
This is not the ideal interface - user shouldn't need to wrap the result in an pa.array of one row.

This is currently implemented this way to reuse the code to pass the result back to C++ code in an object that is well understood (array).

Reply:
Fixed